heltour/tournament/static/lib/js/sort-tables.js

```diff
@@ -0,0 +1,30 @@
+const getCellValue = (tr, idx) =>
+  tr.children[idx].innerText || tr.children[idx].textContent
+
+const comparer = (idx, asc) => (a, b) =>
```

Comment:
I find this to be hard to follow. I'd prefer to have it split out into a few functions that are then put back together:
```js

const cellValue = (tr, idx) =>
  tr.children[idx].innerText || tr.children[idx].textContent
const numericCompare = (a, b) => a - b // implicitly converts to number
const stringCompare = (a, b) => a.toString().localeCompare(b)
const nonEmpty = a => a !== ''
const isNumber = a => nonEmpty(a) && !isNan(a)
const compare = (a, b) => [a,b].every(isNumber) ? numericCompare(a, b) : stringCompare(a, b)
const direction = asc => asc ? 1  :  -1
const cellCompare = (idx, asc) => (a, b) => compare(cellValue(b, idx), cellValue(b, idx)) * direction(asc)

```